tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -105,14 +103,48 @@ const DAY_IN_MS = 24 * 1000 * 60 * 60;
         text-anchor: middle;
       }
 
+      .x-axis .ticks {
+        -webkit-mask-image: linear-gradient(
+          to right,
+          #0000 0%,
+          #000 10%,
+          #000 90%,
+          #0000 100%
+        );
+        mask-image: linear-gradient(
+          to right,
+          #0000 0%,
+          #000 10%,
+          #000 90%,
+          #0000 100%
+        );
+      }
+
       .y-axis {
-        flex-direction: row;
+        flex-direction: row-reverse;
```

Comment:
Just checking, is this "-reverse" needed?

Reply:
Yup, in case of y-axis, the text should appear before the line while for, x-axis, the line should appear before (on top of) the text. Since, DOM order wise, the line is placed before the text, we need the row-reverse.